### PR TITLE
Fix auto bumper

### PR DESCRIPTION
**Pull request by chaodaiG**
Fixes: #2012 

Prow images auto bumper job requires source code of knative/test-infra, it doesn't really make sense to run it in a custom built container. Run this job in regular Prow base image to make it work

/assign chizhg
/cc chizhg

### CODE REVIEW

File: config/prod/prow/jobs/custom-jobs.yaml [hunk 1]
```diff
@@ -212,17 +212,25 @@ periodics:
...
   cluster: "build-knative"
+  extra_refs:
+  - org: knative
+    repo: test-infra
+    base_ref: master
+    path_alias: knative.dev/test-infra
   spec:
     containers:
-    - image: gcr.io/knative-tests/test-infra/prow-auto-bumper:latest
+    - image: gcr.io/knative-tests/test-infra/prow-tests:stable
```
Comment (chizhg) [inline config/prod/prow/jobs/custom-jobs.yaml:222]: Use `prow-tests-go114`?

File: config/prod/prow/jobs/custom-jobs.yaml [hunk 2]
```diff
@@ -212,17 +212,25 @@ periodics:
...
-      - "/prow-auto-bumper"
+      - "go"
       args:
+      - "run"
+      - "./tools/prow-auto-bumper/."
       - "--github-account=/etc/prow-auto-bumper-github-token/token"
       - "--git-userid=knative-prow-updater-robot"
       - "--git-username='Knative Prow Updater Robot'"
       - "--git-email=knative-prow-updater-robot@google.com"
+      - "--dry-run"
```
Comment (chizhg) [inline config/prod/prow/jobs/custom-jobs.yaml:233]: Remove this?

File: tools/prow-auto-bumper/file.go [hunk 3]
```diff
@@ -93,7 +93,7 @@ func (pv *PRVersions) updateAllFiles(fileFilters []*regexp.Regexp, imageFilter *
 	dryrun bool) ([]string, error) {
 	var msgs []string
 	if err := helpers.ChdirToRoot(); err != nil {
-		return msgs, fmt.Errorf("failed to change to root dir")
+		return msgs, fmt.Errorf("failed to change to root dir: '%v'", err)
```
Comment (chizhg) [inline tools/prow-auto-bumper/file.go:96]:
```suggestion
		return msgs, fmt.Errorf("failed to change to root dir: '%w'", err)
```

Comment (chaodaiG) [reply tools/prow-auto-bumper/file.go:96]: Wasn't %w deprecated in go1.14?

File: config/prod/prow/jobs/custom-jobs.yaml (hunk 2 above)
Comment (chaodaiG) [reply config/prod/prow/jobs/custom-jobs.yaml:233]: ah..

File: config/prod/prow/jobs/custom-jobs.yaml (hunk 1 above)
Comment (chaodaiG) [reply config/prod/prow/jobs/custom-jobs.yaml]: done

File: config/prod/prow/jobs/custom-jobs.yaml (hunk 2 above)
Comment (chaodaiG) [reply config/prod/prow/jobs/custom-jobs.yaml]: fixed

File: tools/prow-auto-bumper/file.go (hunk 3 above)
Comment (chizhg) [reply tools/prow-auto-bumper/file.go:96]: I heard from you that it's deprecated, but actually seems not? There are still a lot of `%w` in knative/pkg

Comment (chaodaiG) [reply tools/prow-auto-bumper/file.go:96]: It was failing when we first tried out go1.14, but in beta, might have been fixed

Comment (chaodaiG) [reply tools/prow-auto-bumper/file.go]: done

File: config/prod/prow/jobs/custom-jobs.yaml [hunk 4]
```diff
@@ -212,13 +212,20 @@ periodics:
...
     containers:
-    - image: gcr.io/knative-tests/test-infra/prow-auto-bumper:latest
+    - image: gcr.io/knative-tests/test-infra/prow-tests-go114:stable
       imagePullPolicy: Always
       command:
-      - "/prow-auto-bumper"
+      - "go"
       args:
+      - "run"
+      - "./tools/prow-auto-bumper/."
```
Comment (chizhg) [inline config/prod/prow/jobs/custom-jobs.yaml:228]: It's the first time I've seen path like this, but seems it does work 😅 

/lgtm

**Pull request merged**